F21/mahdirahbar/lower.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+
+# lower
+# Developers: Mahdi Rahbar
+# License: -
+
+from langdetect import detect, DetectorFactory
+
+
+class Lower: 
+    def __init__(self, string, lang):
+        self.string = string 
+        self.lang = lang 
+        self.detected_lang = self.lang_detector()
+        self.word_list = self.string_splitter(string)
+
+    def string_splitter(self, string):
```

Comment:
This does exactly what you want it to do, but for future reference Python has a built in function `string.split()` that would do the same for you.